typescript/web/src/components/spinner/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Spinner as ChakraSpinner, ThemingProps } from "@chakra-ui/react";
+
+type SpinnerProps = {
+  id?: string | undefined;
+  color?: string | undefined;
+  ariaLabel?: string | undefined;
+  size?: ThemingProps<"Spinner">["size"] | undefined;
+  visibility?: "visible" | "hidden" | undefined;
+  key?: string | undefined;
+  ref?: React.Ref<HTMLDivElement> | undefined;
+};
+export const Spinner = ({
+  id = undefined,
+  color = "black",
+  ariaLabel = "",
+  size = "xl",
+  visibility = "visible",
+  key = undefined,
+  ref = undefined,
+}: SpinnerProps) => {
+  return (
+    <ChakraSpinner
+      id={id}
+      aria-label={ariaLabel}
+      thickness="4px"
```

Comment:
Do we really need to override Chakra default thickness?

Reply:
No, not necessarily. I reuse the value of Nico's example, but I guess default value will work just fine.